docs/guides/cloud/integrations/data-extract-api.mdx

```diff
@@ -47,8 +47,20 @@ the type of data that will be retrieved.
 
 https://cloud.cypress.io/enterprise-reporting/report?token=YOUR-API-KEY&report_id=REPORT-TYPE&export_format=csv
 
-Query parameters include a mandatory `start_date` and optional `end_date` value. For example,
-`?start_date=2024-01-01&end_date=2024-04-15`
+The following query parameters are available for use:
+
+- start_date. **Required** and should be passed in either yyyy-MM-dd or yyyy-MM-ddTHH:mm:ss format.
+
+  For example, `?start_date=2024-01-01` or `?start_date=2024-01-01T12%3A30%3A00`
+
+- end_date. Optional and should be passed in either yyyy-MM-dd or yyyy-MM-ddTHH:mm:ss format.
+
+  For example, `?end_date=2024-01-01` or `?end_date=2024-01-01T12%3A30%3A00`
+
+- projects. Optional, and accepts the name of your Cypress project. The project name should be URL encoded. You can
+  pass as many projects parameters as needed.
+
+  For example, `&projects=project%20a&projects=test123`
```

Comment:
It’d be nice to show the desired state of these without the encoding and everything. Like “For example, if you want to see reporting for projects with the name Dashboard and design-system, use the following query param”. Otherwise I’m trying to parse what the desired state was for the time formats and the project names from just the url and removing the encoding, but I didn’t even know what the names of the projects were to begin with.

Reply:
incorporated.  Coming in the next commit